### ISSUE TITLE
App reloads on rotation and crashes after onStop()

### ISSUE THREAD

**Issue by easyaspi314** [pylerSM/XInstaller#26]
There are a few general easy-to-fix app flaws.

The rotation reloading the UI is easy, just add 

```
android:configChanges="orientation|screenSize"
```

to the XML tag of each Activity.

The crash is also a simple one. All you have to do is check if whatever it is complaining about is `null` and recreate it if so.
__What matters:__

```
10-16 23:01:07.995 E/AndroidRuntime(28513): Caused by: java.lang.NullPointerException
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at com.pyler.xinstaller.Preferences$Settings.onCreate(Preferences.java:43)
```

Full LogCat

```
10-16 23:01:07.995 E/AndroidRuntime(28513): FATAL EXCEPTION: main
10-16 23:01:07.995 E/AndroidRuntime(28513): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.pyler.xinstaller/com.pyler.xinstaller.Preferences}: java.lang.NullPointerException
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2110)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2135)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.ActivityThread.access$700(ActivityThread.java:143)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1241)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.os.Looper.loop(Looper.java:137)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.ActivityThread.main(ActivityThread.java:4950)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at java.lang.reflect.Method.invokeNative(Native Method)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at java.lang.reflect.Method.invoke(Method.java:511)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1004)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:771)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at dalvik.system.NativeStart.main(Native Method)
10-16 23:01:07.995 E/AndroidRuntime(28513): Caused by: java.lang.NullPointerException
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at com.pyler.xinstaller.Preferences$Settings.onCreate(Preferences.java:43)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:796)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1035)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1017)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.FragmentManagerImpl.dispatchCreate(FragmentManager.java:1799)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.Activity.onCreate(Activity.java:921)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at com.pyler.xinstaller.Preferences.onCreate(Preferences.java:30)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.Activity.performCreate(Activity.java:5177)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1094)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2074)
10-16 23:01:07.995 E/AndroidRuntime(28513): 	... 12 more
```

**Comment by pylerSM** [comment]
Thank you. I will add those tags for Settings Activity.

**Comment by easyaspi314** [comment]
Fixed in #27. I also added a few summaries.
